t/cobrand/northumberland.t

```diff
@@ -111,4 +111,84 @@ FixMyStreet::override_config {
+    MAPIT_URL => 'http://mapit.uk/',
+}, sub {
+    my $superuser = $mech->create_user_ok(
+        'superuser@example.com',
+        name         => 'Super User',
+        is_superuser => 1,
+    );
+    $mech->log_in_ok( $superuser->email );
+
+    for my $host ( 'northumberland', 'fixmystreet' ) {
+        ok $mech->host($host), "change host to $host";
+
+        my ($problem_to_update) = $mech->create_problems_for_body(
+            1,
+            $body->id,
+            'Test',
+            { cobrand => $host },
+        );
+
+        subtest "User assignment on $host site" => sub {
+            $mech->get_ok( '/report/' . $problem_to_update->id );
+            $mech->click_ok('.btn--shortlist');
+            my $comment
+                = $problem_to_update->comments->order_by('-id')->first;
+            is_deeply $comment->get_extra_metadata, {
+                shortlisted_user => $superuser->email,
+                is_superuser     => 1,
+            }, 'Comment created for user assignment';
+
+            $mech->get_ok( '/report/' . $problem_to_update->id );
+            $mech->click_ok('.btn--shortlisted');
+            $comment
+                = $problem_to_update->comments->order_by('-id')->first;
+            is_deeply $comment->get_extra_metadata, {
+                shortlisted_user => undef,
+                is_superuser     => 1,
+            }, 'Comment created for user un-assignment';
+        };
+
+        subtest "Extra details on $host site" => sub {
+            $mech->get_ok( '/report/' . $problem_to_update->id );
+            $mech->submit_form(
+                button  => 'save',
+                form_id => 'report_inspect_form',
+                fields  => {
+                    detailed_information => 'A',
+                    include_update       => 0,
+                },
+            );
+            my $comment
+                = $problem_to_update->comments->order_by('-id')->first;
+            is_deeply $comment->get_extra_metadata, {
+                detailed_information => 1,
+                is_superuser         => 1,
+            }, 'Comment created for extra details';
+
+            $mech->get_ok( '/report/' . $problem_to_update->id );
+            $mech->submit_form(
+                button  => 'save',
+                form_id => 'report_inspect_form',
+                fields  => {
+                    detailed_information => '',
+                    include_update       => 0,
+                },
+            );
+            $comment
+                = $problem_to_update->comments->order_by('-id')->first;
+            is_deeply $comment->get_extra_metadata, {
+                detailed_information => 1,
+                is_superuser         => 1,
+            }, 'Comment created for extra details unsetting';
+            is_deeply $problem_to_update->get_extra_metadata,
+                {},
+                'Extra details unset on problem';
+        };
+    }
```

Comment:
Would be good to have a test here of sending an update sets the appropriate Open311 attribute fields okay.